### PR TITLE
[MIG] event_registration_cancel_reason: Migration to v12

### PR DESCRIPTION
**Pull request by victor-ditay**

### CODE REVIEW

File: event_registration_cancel_reason/views/event_registration_view.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         <field name="inherit_id" ref="event.view_registration_search"/>
         <field name="arch" type="xml">
             <filter context="{'group_by':'state'}" position="after">
```
Comment (pedrobaeza) [inline event_registration_cancel_reason/views/event_registration_view.xml]: Can we depend on a filter with name for securing the inheritance?

**Review by pedrobaeza — APPROVED** [review]

**Comment by victor-ditay** [comment]
@pedrobaeza Changes done

File: event_registration_cancel_reason/views/event_registration_view.xml [hunk 2]
```diff
@@ -22,8 +22,8 @@
         <field name="model">event.registration</field>
         <field name="inherit_id" ref="event.view_registration_search"/>
         <field name="arch" type="xml">
-            <filter context="{'group_by':'state'}" position="after">
-                <filter string="Cancellation reason" domain="[]" context="{'group_by': 'cancel_reason_id'}"/>
+            <filter name="status" context="{'group_by':'state'}" position="after">
```
Comment (pedrobaeza) [inline event_registration_cancel_reason/views/event_registration_view.xml]: Then remove `context` attribute here in the selector.

Comment (victor-ditay) [reply event_registration_cancel_reason/views/event_registration_view.xml]: Done @pedrobaeza

**Comment by rafaelbn** [comment]
Please @victormmtorres review UX in the line of Odoo

**Comment by victor-ditay** [comment]
Changes done @rafaelbn

File: event_registration_cancel_reason/README.rst [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+Authors
+~~~~~~~
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `Tecnativa <https://www.tecnativa.com>`__:
+
```
Comment (rafaelbn) [inline event_registration_cancel_reason/README.rst:86]: I've been reviewing this module from the 1st version. I think I'm a contributor :smile:

**Review by rafaelbn — APPROVED** [review]
Functional test :+1:

**Comment by victor-ditay** [comment]
Changes done @rafaelbn

**Pull request closed without merging**